RGBA methods for hex strings

billburgess opened the pull request:
I deprecated the old hex string methods and duplicated them with an RGBA designation. I updated the method descriptions to indicate that the alpha goes in the end positions. This will avoid breaking any existing users out there that may be using the alpha in first position. Tests also updated to use the new RGBA method and pass accordingly.

mRs- commented:
wow thanks! That's neat!

mRs- commented:
Could you hand in new tests for the new methods, so we can leave the old ones alone? Because of the code coverage :)

billburgess commented:
Sure. Give me a few and I'll update my pull request.

billburgess commented:
I reverted the existing tests and duplicated against the new RGBA changes. I also added a new test to cover full length RGBA strings to get full code coverage. Should be good to go.
